lib/ui/profile/usecases/set_bio_use_case.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:hypha_wallet/core/error_handler/model/hypha_error.dart';
+import 'package:hypha_wallet/core/network/api/aws_amplify/amplify_service.dart';
+import 'package:hypha_wallet/ui/architecture/result/result.dart';
+
+class SetBioUseCase {
+  final AmplifyService _amplifyService;
+
+  SetBioUseCase(this._amplifyService);
+
+  Future<Result<bool, HyphaError>> run(String name) async {
+    try {
+      print('set bio to $name');
+      // ignore: unused_local_variable
```

Comment:
all these unused local variables are because we return dynamic from the call. so we dont know what to do with the result